content/enterprise_influxdb/v1.9/administration/configure/config-data-nodes.md

```diff
@@ -512,9 +512,13 @@ Default is `false`.
 
 By default, this option is set to false and writes are processed in the order that they are received. This means if any points are in the hinted handoff (HH) queue for a shard, all incoming points must go into the HH queue.
 
-If true, writes may process in a different order than they were received. This can reduce the time required to drain the HH queue and increase throughput during recovery.
+If set to `true`, writes may process in a different order than they were received. This can reduce the time required to drain the HH queue and increase throughput during recovery.
 
-**Do not enable if your use case involves updating points, which may cause points to be overwritten.** To overwrite an existing point, the measurement name, tag keys and values (if the point includes tags), field keys, and timestamp all have to be the same as a previous write.
+Whn a remote write fails, the database will send data to the hinted handoff (HH) queue for a short period of time before allowing incoming remote writes to be retried again. This time period grows with each failed write. The greater the number of successive write failures, the longer a node will delay writes before retrying them. Consecutive write failures will cause increased delay times before retrying writes in both emptying HH queues and in remote writes.
```

Comment:
"... before retrying them, up to a maximum set by the configuration parameter `retry-max-interval`.  Consecutive..."

Reply:
@davidby-influx I'll set up a quick meeting if needed - are you saying that this is not the place to discuss  throttled writes?  Also, if there is another parameter that needs to be configured as part of this, then it should be addressed as part of this issue.

Reply:
@davidby-influx can you review changes made and let me know if we need a quick Zoom to add any updates?  Thank you!